components/timezone/src/provider/names.rs

```diff
@@ -95,6 +95,8 @@ pub struct IanaToBcp47MapV3<'data> {
     // Note: this is 9739B as `ZeroVec<TimeZoneBcp47Id>` (`ZeroVec<TinyStr8>`)
     // and 9335B as `VarZeroVec<str>`
     pub bcp47_ids: ZeroVec<'data, TimeZoneBcp47Id>,
+    /// Metazones
+    pub metazones: ZeroVec<'data, u16>,
```

Comment:
Issue: Please explain/document this field. Previous authors of this data struct (mostly me) have gone above and beyond documenting how the pieces fit together, since it is non-obvious. I don't immediately see how a `ZeroVec<u16>` relates to metazones.